aiken.toml

```diff
@@ -1,6 +1,9 @@
 name = "aiken-lang/trees"
-version = "0.0.0"
+version = "0.0.1"
 licences = ["Apache-2.0"]
 description = "We got trees!"
 
-dependencies = []
+dependencies = [
+  { name = "aiken-lang/stdlib", version = "1e82ccd5d30d68615b6c0f3b84947b5e93f23549", source = "github" },
+  { name = "Cardano-Fans/acca", version = "1be26cd7f93d009d837b3d446be4c7f09c83b336", source = "github" }
```

Comment:
Please do not add aiken external deps

Reply:
Should I incline all functions / types here in this project which are needed, e.g. some new file like: inline.ak ?

Reply:
Are you referring to this implementation of Either (https://github.com/Cardano-Fans/acca/blob/main/lib/acca/either.ak) if yes why dop you consider this hacky? Either is right aligned but I don't think this is hacky, hacky by which standards, code style or what?

Reply:
No, I am referring to the Haskell implementation of the Merkle Trees, and in particular the type for the Proof. Right now, it's a type-alias  is a list of `Either`, because I was lazy and didn't define a proper type for it... It works, it's just not very elegant.

Reply:
Thank You for clarification.